src/CodeCoverage/Collector.php

```diff
@@ -30,9 +30,9 @@ class Collector
 	public static function detectEngines(): array
 	{
 		return array_filter([
-			extension_loaded('pcov') ? self::ENGINE_PCOV : null,
-			defined('PHPDBG_VERSION') ? self::ENGINE_PHPDBG : null,
-			extension_loaded('xdebug') ? self::ENGINE_XDEBUG : null,
+			extension_loaded('pcov') ? [self::ENGINE_PCOV, phpversion('pcov')] : null,
+			defined('PHPDBG_VERSION') ? [self::ENGINE_PHPDBG, PHPDBG_VERSION] : null,
+			extension_loaded('xdebug') ? [self::ENGINE_XDEBUG, phpversion('xdebug')] : null,
```

Comment:
maybe `[self::ENGINE_XDEBUG => phpversion('xdebug')]`?

Reply:
It would be harder to work with.

Reply:
i thought it would be easier xd